djstripe/models.py

```diff
@@ -45,12 +45,41 @@ def convert_tstamp(response, field_name=None):
 
 
 class StripeObject(TimeStampedModel):
+    # This must be defined in descendants of this model/mixin
+    # e.g. "Event", "Charge", "Customer", etc.
+    stripe_api_name = None
 
     stripe_id = models.CharField(max_length=50, unique=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
"old style class" versus "new style class".  My IDE highlights these so I correct them as I move along. I've never looked into it much. But here are a few things that pop up:

https://code.djangoproject.com/ticket/7342
http://stackoverflow.com/questions/2005150/why-is-djangos-meta-an-old-style-class

This is arbitrary for me, so I'll happily defer to your high level direction